app/views/users/schedule_student.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Student Scheduling Page</h1>
+<%= button_to "Cancel", {:action => 'show'}, :method => :get %>
+<p>Please select which session you would like to attend</p>
+<% currUser = User.find_by(email: "john@tamu.edu") #temporary hard coded value to make this function with fake data %>
+<table>
+  <tr>
+    <th>Tutor Name</th>
+    <th>Contact Info</th>
+    <th>Scheduled Datetime</th>
+  </tr>
+  <% @sessions.each do |session| %>
+    <!--
+      TODO: Make sure the date is later than now and that the status is not cancelled
+    -->
+    <tr>
+      <!-- Make a row in the table for each tutoring session. Includes details and join option -->
+      <% user = User.find(session.tutor_id) %>
+      <td><%= user.first_name + " " + user.last_name %></td>
+      <td><%= user.email %></td>
+      <% date = session.scheduled_datetime.localtime.strftime("%B %d at %l:%M %p %Z") %>
+      <td><%= date %></td>
+      <!--
+        TODO: Pass user that was passed in to button instead of hard coded value
+            ex: {:action => 'schedule', :id => session.id, :user => params[:user]}
+      -->
+      <td><%= button_to "Join session", {:action => 'schedule_session_student', :sessionID => session}, :method => :post %></td>
+    </tr>
+  <% end %>
+</table>
```

Comment:
Make sure to add a newline at the end of the file, as-per coding standards

Reply:
Added the newline in the newest commit